Build/Build.cs

```diff
@@ -3,24 +3,63 @@
 using System.Linq;
 using LibGit2Sharp;
 using Nuke.Common;
+using Nuke.Common.CI.GitHubActions;
 using Nuke.Common.Execution;
 using Nuke.Common.Git;
 using Nuke.Common.IO;
 using Nuke.Common.ProjectModel;
 using Nuke.Common.Tooling;
+using Nuke.Common.Tools.CoverallsNet;
 using Nuke.Common.Tools.DotNet;
 using Nuke.Common.Tools.GitVersion;
 using Nuke.Common.Tools.ReportGenerator;
 using Nuke.Common.Tools.Xunit;
 using static Nuke.Common.IO.FileSystemTasks;
 using static Nuke.Common.IO.PathConstruction;
+using static Nuke.Common.Tools.CoverallsNet.CoverallsNetTasks;
 using static Nuke.Common.Tools.DotNet.DotNetTasks;
 using static Nuke.Common.Tools.ReportGenerator.ReportGeneratorTasks;
 using static Nuke.Common.Tools.Xunit.XunitTasks;
 using static Serilog.Log;
 
 [UnsetVisualStudioEnvironmentVariables]
 [DotNetVerbosityMapping]
+[CustomGitHubActions(
```

Comment:
We had an internal chat and we do like most of the changes, but we like to stay away from the code generation part. Although we don't like YAML, the generation code isn't much prettier and will make it more difficult to understand what's going on.

Reply:
If you don't like to see the generation code, just put a `partial class Build {}` in a separate file and decorate this one ;)

So you don't see the yaml AND the generation code in future :D